hibernate-core/src/main/java/org/hibernate/service/ServiceRegistry.java

```diff
@@ -7,7 +7,22 @@
 package org.hibernate.service;
 
 /**
- * A registry of {@linkplain Service services}.
+ * A registry of {@linkplain Service services}. This interface abstracts
```

Comment:
1. You are just listing known implementations.  Javadoc already does this for you.
2. In general, not a fan of linking the same thing multiple times within the same javadoc.  Here it is not as terrible as doing it within the actual javadoc text as `@see` is rendered separately.  But it still adds no value imo